Java/parkservices/src/main/java/com/amazon/randomcutforest/extendedrandomcutforest/threshold/BasicThresholder.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.randomcutforest.extendedrandomcutforest.threshold;
+
+import static com.amazon.randomcutforest.CommonUtils.checkArgument;
+
+import com.amazon.randomcutforest.extendedrandomcutforest.threshold.state.BasicThresholderState;
+
+public class BasicThresholder implements IThresholder {
+
+    public static int IS_ANOMALY = 1;
+
+    public static int NOT_ANOMALY = 0;
```

Comment:
Maybe I missed it, but I don't see these constants being used.

Reply:
Removed